tests/dualtor/test_standby_tor_upstream_mux_toggle.py

```diff
@@ -20,7 +20,7 @@
                             'run_garp_service', 'run_icmp_responder')
 ]
 
-PAUSE_TIME = 10
+PAUSE_TIME = 90
```

Comment:
why extend pause_time?

Reply:
The crm_facts collection take around 100 Pause_time so we have increased the PAUSE_TIME to 90. Which worked properly in multiple trials.